portfolio-project-matching-app/pages/myProjects.js

```diff
@@ -35,12 +35,12 @@ const myProjects = () => {
     }, [])
 
     return (
-        <div>
+        <div className='background'>
             {authUser.user ? 
             projects.map((project) => {
                 return (
                     <div key={project.id} className='p-2'>
-                        <ProjectCard project={project} />
+                        <ProjectCard initialProject={project} />
```

Comment:
Forgot to update this. I'm wondering whether I actually did update this and maybe messed up while resolving merge conflicts.